bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/Proposal.scala

```diff
@@ -74,12 +76,16 @@ object Proposal {
     List.empty[Keyword],
     Investigators.empty,
     List.empty[Target],
-    List.empty[Observation],
+    List(Observation.empty),
     ProposalClass.empty,
     currentSchemaVersion)
 
   def apply(m:M.Proposal) = new Proposal(m)
 
+  // REL-3290: Create a non-empty list of observations, i.e. make sure that if there are no defined observations,
+  // then an empty one is provided to serve as a dummy observation for users to edit.
+  def nonEmptyObsList(olist: List[Observation]): List[Observation] =
```

Comment:
Maybe we should use `NonEmptyList` here

Reply:
I did try that at first, actually, but `NonEmptyList` seems to be missing a bunch of convenient methods (from what I could tell) that broke a lot of code.